app/controllers/my_facilities_controller.rb

```diff
@@ -57,6 +57,10 @@ def csv_maker
     send_data csv_data, type: "text/csv", filename: filename
```

Comment:
Hey I think here is where you will want to queue the download job. Something like `MyFacilitiesDownloadJob.perform_async`